### PR TITLE
LPS-158055 Lower opacity of the rest of elements when dropping

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/Topper.js [hunk 1]
```diff
@@ -177,6 +178,23 @@ function TopperContent({
...
+
+		if (
+			!itemIsAncestor(highlightedItem, item, layoutData) &&
+			item.type === LAYOUT_DATA_ITEM_TYPES.fragment
+		) {
+			return true;
+		}
+
+		return false;
+	}, [layoutData, highlightedItemId, isDraggingSource, item]);
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/Topper.js:196]: I know this was here before, but could you check if replacing this layoutData dependency with `useSelectorCallback` makes any performance improvement? (in a separated task, of course :heart:)

**Review by p2kmgcl — COMMENTED** [review]
Looks awesome, but I think I found a bug :sweat_smile: 

Do I have to enable some feature flag to test it?

image

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/Topper.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/Topper.js]: Finally removed this code, so it's not necessary 🥰

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by ozysouza** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**